stackit/services/dns/zone/resource_test.go

```diff
@@ -306,22 +305,18 @@ func TestToPayloadUpdate(t *testing.T) {
 				Type:              types.StringValue("Type"),
 				ContactEmail:      types.StringValue("ContactEmail"),
 				PrimaryNameServer: types.StringValue("PrimaryNameServer"),
-				Primaries: types.ListValueMust(types.StringType, []attr.Value{
-					types.StringValue("Primary"),
-				}),
-				RetryTime:     types.Int64Value(3),
-				RefreshTime:   types.Int64Value(4),
-				ExpireTime:    types.Int64Value(5),
-				DefaultTTL:    types.Int64Value(4534534),
-				NegativeCache: types.Int64Value(-4534534),
-				IsReverseZone: types.BoolValue(true),
+				RetryTime:         types.Int64Value(3),
+				RefreshTime:       types.Int64Value(4),
+				ExpireTime:        types.Int64Value(5),
+				DefaultTTL:        types.Int64Value(4534534),
+				NegativeCache:     types.Int64Value(-4534534),
+				IsReverseZone:     types.BoolValue(true),
```

Comment:
Shoud we instead leave the field here in the input? As that's what is actually happening on the resource right? Even though we then still expect the output to be `nil`

Reply:
Yes, makes sense. Fixed it